src/v0/index.ts

```diff
@@ -35,10 +35,12 @@ async function streamToArrayBuffer(
       break;
     }
 
-    const newResult = new Uint8Array(result.length + value.length);
-    newResult.set(result);
-    newResult.set(value, result.length);
-    result = newResult;
+    if (value) {
```

Comment:
Older TS compilers were erroring on the `value.length` reference below because the type says `value` could be undefined. I don't think it's a realistic case you would encounter in runtime, though due to the break on done condition above.